content/en/blog/_posts/image-compatibility-in-cloud-native-environments/index.md

```diff
@@ -0,0 +1,159 @@
+It is common practice to require the use of specific versions of the kernel, its configuration, device drivers, or system components.
+Despite the existence of the [Open Container Initiative (OCI)](https://opencontainers.org/), a governing community to define standards and specifications for container images, there has been a gap in expression of such compatibility requirements.
+The need to address this issue has led to different proposals and, ultimately, an implementation in Kubernetes' [Node Feature Discovery (NFD)](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html).
+
+[NFD](https://kubernetes-sigs.github.io/node-feature-discovery/stable/get-started/index.html) is an open source Kubernetes project that automatically detects and reports [hardware and system features](https://kubernetes-sigs.github.io/node-feature-discovery/v0.17/usage/customization-guide.html#available-features) of cluster nodes. This information helps users to schedule workloads on nodes that meet specific system requirements, which is especially useful for applications with strict hardware or operating system dependencies.
+
+## The need for image compatibility specification
+
+### Dependencies between containers and host OS
+
+A container image is built on a base image, which provides a minimal runtime environment, often a stripped-down Linux userland, completely empty or distroless. When an application requires certain features from the host OS, compatibility issues arise. These dependencies can manifest in several ways:
+
+- **Drivers**:
+  Host driver versions must match the supported range of a library version inside the container to avoid compatibility problems. Examples include GPUs and network drivers.
+- **Library or Software**:
+  The container must come with a specific version or range of versions for a library or software to optimally run in the environment. Examples from high performance computing are MPI, EFA, or Inifiniband.
+- **Kernel Modules or Features:**:
+  Specific kernel features or modules must be present. Examples include having support of write protected huge page faults, or the presence of VFIO
+- **And more…**
+
+While containers in Kubernetes are the most likely unit of abstraction for these needs, the definition of compatibility can extend further to include other container technologies such as Singularity, and other OCI artifacts such as binaries from a spack binary cache.
+
+### Multi cloud and hybrid cloud challenges
+
+Containerized applications are deployed across various Kubernetes distributions and cloud providers, where different host operating systems introduce compatibility challenges.
+Often those have to be pre-configured before workload deployment or are immutable.
+For instance, different cloud providers will include different operating systems like:
+
+- **RHCOS/RHEL**
+- **Photon OS**
+- **Amazon Linux 2**
+- **Container-Optimized OS**
+- **Azure Linux OS**
+- **And more...**
```

Comment:
```suggestion
  - **RHCOS/RHEL**
  - **Photon OS**
  - **Amazon Linux 2**
  - **Container-Optimized OS**
  - **Azure Linux OS**
  - **And more...**
```

2 spaces

Reply:
Why do we need 2 spaces here? If I am not mistaken, according to MD007 unordered list indentation, the rule of 2 spaces applies to sublists only.

Reply:
> I wouldn't put "And more…" in bold

Fixed as well.